src/common/snippets/include/snippets/op/load.hpp

```diff
@@ -42,6 +42,18 @@ class Load : public ngraph::op::Op {
     size_t m_count = 0lu;
 };
 
+// todo: LoadReshape is just Load (and mapped on LoadEmitter), it just allows to keep correct shape propagation
+//  when we decompose Transpose to Load and Store.
+class LoadReshape : public Load {
```

Comment:
If we use order as parameter, node should be called as `LoadTranspose`. But honestly the node transposes nothing, just reshapes output. Can we set `output_shape` instead of `order`?

Or probably we can insert inplaced `Reshape` node between `Load` and `Store` in `TransposeDecomposition`. Inplace `Reshape` is `Reshape` with the same input and output tensors (for `AssignRegister`). But we should cover `Constant` of `Reshape` in `AssignRegister` as well.... Let's discuss it

Reply:
todo: create Reshape ticket

Reply:
name: snippets::op::Reshape()